app/scripts/utils/interval-tree.js

```diff
@@ -75,7 +75,7 @@ function addHelper(node, interval) {
    * @public
    * @param {Array} intreval Array with start and end points of the interval.
    */
-IntervalTree.prototype.add = function (interval) {
+IntervalTree.prototype.add = function add(interval) {
```

Comment:
Would it make sense to use es6 syntax here? `(interval) => ...` rather than creating a named function which won't be called by name?

Reply:
Arrow functions are great for stateless / pure function as they cannot have any `this` state. But since `IntervalTree` is a factory (or class) using an arrow function here would be odd because then a member function can't access it's own state. If it doesn't need any state they I would argue it shouldn't be a member function.

Adding the name here makes sense for debugging purposes I believe.

Reply:
Yeah, the debugging help is the rationale the eslint docs give. I leave these as they are.